Sources/Afterpay/Resources/IntroText.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  IntroText.swift
+//  Afterpay
+//
+//  Created by Scott Antonac on 25/8/21.
+//  Copyright © 2021 Afterpay. All rights reserved.
+//
+
+import Foundation
+
+public enum AfterpayIntroText: String {
+  case NONE = ""
+  case make = "make "
+  case makeTitle = "Make "
+  case pay = "pay "
+  case payTitle = "Pay "
+  case `in` = "in "
+  case inTitle = "In "
+  case or = "or "
+  case orTitle = "Or "
+  case payIn = "pay in "
+  case payInTitle = "Pay in "
+}
```

Comment:
What do you think of rather than having the white space here, keeping it in the `fourPaymentsFormat`; to cope with the `.none` you'd then trim any extra with `.trimmingCharacters(in: .whitespaces)` on the `String(format:)` call.

Reply:
I think that's a much cleaner solution. Means doesn't need to be added for any new options that may be required later.

Reply:
Changed in 0a4e72a

Reply:
Nice one @huwr. I should have thought to suggest that for the Android solution too.